standalone/src/recording/index.tsx

```diff
@@ -142,6 +126,23 @@ async function wsHandlerRecording(store: Store, ev: WebSocketMessage<WebsocketEv
 
         break;
     }
+    case `custom_${pluginId}_call_state`: {
+        const data = ev.data as CallStateData;
+
+        const profiles = profilesInCallInChannel(store.getState(), data.channel_id);
```

Comment:
Yeah, I had to tweak a couple of things to make sure that was the case, the key point would be this:

https://github.com/mattermost/mattermost-plugin-calls/blob/dee47eeaff59554d21d0c41f7edf5b1d037cb213/standalone/src/init.ts#L267

Reply:
Ah ok, that's good. But just to double check -- if the call state includes the user list, is it guaranteed to be up to date at this point? If so, could we use that instead of relying on the store's state?

Reply:
The call state only includes the user ids of the participants. We still need to potentially fetch the profiles. But yes, it's guarantee to be up to date once sent because we are sending the event with the call mutex locked on the server. Any subsequent change (e.g. join/leave) will new events which are guaranteed to come after the initial one.